style.css

```diff
@@ -43,24 +43,205 @@ nav {
   position: absolute;
   right: 1rem;
   display: none;
+  top: 1.5rem;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 21px;
+}
+
+.close {
+  display: none;
 }
 
 @media (max-width: 500px) {
   .menu-button {
     display: flex;
   }
 
+  .close-button {
+    width: 30px;
+    height: 30px;
+    left: 5px;
+    top: 5px;
+  }
+
+  .close {
+    display: flex;
+    height: 50px;
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓